tls/s2n_client_extensions.c

```diff
@@ -218,8 +217,13 @@ int s2n_client_extensions_recv(struct s2n_connection *conn, struct s2n_array *pa
     return 0;
 }
 
-static int s2n_recv_client_server_name(struct s2n_connection *conn, struct s2n_stuffer *extension)
+int s2n_parse_client_hello_server_name(struct s2n_connection *conn, struct s2n_stuffer *extension)
 {
+    if (strlen(conn->server_name) != 0) {
```

Comment:
as server_name is fixed length, how about:

```
if (conn->server_name[0]) {
```

Reply:
Thanks for the review, update.